client/src/components/Token.vue

```diff
@@ -5,18 +5,19 @@
     <br/>
     <TokenChoices :cost='banCost' choice='ban'/>
 
-    <button @click='singleDevice ? singleSubmit : multiSubmit'>Next</button>
+    <button @click='singleSubmit'>Next</button>
   </div>
 </template>
 
 <script>
 import TokensLeft from './Token-TokensLeft.vue'
 import TokenChoices from './Token-Choices.vue'
 import { settingsStore } from '../store/settings.js'
-import { playerChoiceStore } from '../store/player-choices.js'
+// import { playerChoiceStore } from '../store/player-choices.js'
 import { EventBus } from '../main.js'
 
 export default {
+  props: ['decMade'],
```

Comment:
In the even of a conflicting namespace between prop and data, try to be consistent.
ie )
```
props: ['decisionsMade-prop']
```